src/Symfony/Bundle/FrameworkBundle/Tests/Security/SecurityHelperTraitTest.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Security;
+
+use Symfony\Bundle\FrameworkBundle\Security\SecurityHelperTrait;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\User\User;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+class SecurityHelperTraitTest extends TestCase
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The class SecurityHelperTraitTest has 15 methods. Consider refactoring SecurityHelperTraitTest to keep number of methods under 10.](https://www.codacy.com/app/git_9/symfony/file/2332002738/issues/source?bid=3207821&fileBranchId=3207915#l28)